pixi.toml

```
@@ -0,0 +1,153 @@
+[project]
+name = "plateau"
+channels = ["conda-forge"]
+platforms = ["osx-arm64", "osx-64", "linux-64", "linux-aarch64", "win-64"]
+
+[tasks]
+postinstall = "pip install --no-build-isolation --no-deps --disable-pip-version-check -e ."
+
+[dependencies]
+python = ">=3.10"
+dask = ">=2022.5.1"
+decorator = "*"
+msgpack-python = ">=0.5.2"
+numpy = ">1.23,<2"
+pandas = ">1.5.0"
+pyarrow = ">=4.0.1"
```

Drop old Arrow versions. We should only support the last four releases.
```suggestion
pyarrow = ">=14"
```